common/connect.go

```diff
@@ -13,15 +13,15 @@
 package common
 
 import (
-	"net/http"
 	"log"
+	"net/http"
```

Comment:
why isn't this list in alphabetical order like the list below? why are there two separate lists?